AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -912,7 +912,9 @@ - (ASCellNode *)nodeAtIndexPath:(NSIndexPath *)indexPath
       context = completedNodesSection[row];
     }
   }
-  
+  if (_inverted) {
+    context.node.transform = CATransform3DMakeRotation(M_PI, 0, 0, 1.0);
```

Comment:
It might be better to put this code in `-[ASCollectionView/ASTableView dataController:nodeBlockAtIndexPath:]` near the bottom along with the other customization that gets applied to the node, so that it's run exactly once per node, so the data controller doesn't have to be aware of the inversion.